config/php-fpm/php.ini

```diff
@@ -2,3 +2,6 @@
 
 upload_max_filesize = 100M
 post_max_size = 100M
+
+[mail function]
+sendmail_path = /usr/bin/env catchmail -smtp-port 1025 -f allen.moore@10up.com
```

Comment:
@allenmoore I see you've added your email as an argument. Will the setup work with a generic/dummy email address?